core/src/main/scala/org/broadinstitute/dsde/rawls/dataaccess/slick/AttributeComponent.scala

```diff
@@ -269,6 +269,15 @@ trait AttributeComponent {
       }
     }
 
+    def findUniqueStringsByNameQuery(attrName: AttributeName, queryString: String) = {
+      filter(rec =>
+        rec.namespace === attrName.namespace &&
+        rec.name === attrName.name &&
+        rec.valueString.isDefined &&
+        rec.valueString.like(s"%$queryString%")
```

Comment:
Could you show, using the Slick logger, what this query generates now that the `like` is present? I want to make sure we aren't opening ourselves up to sql injection.